sdk/cosmosdb/cosmos/src/client/Item/Item.ts

```diff
@@ -143,9 +144,9 @@ export class Item {
         await this.container.readPartitionKeyDefinition();
       this.partitionKey = extractPartitionKey(body, partitionKeyDefinition);
     }
-
     const err = {};
-    if (!isItemResourceValid(body, err)) {
+    if (!isResourceValid(body, err)) {
+      setDiagnostics(`"ItemDefinition.replace": ${err}`);
```

Comment:
Why do we prefix with ItemDefinition.replace here and not at other places like above readItem call?